packages/pn-pa-webapp/src/main.tsx

```diff
@@ -32,16 +32,16 @@ async function doTheRender() {
 
     root.render(
       <Provider store={store}>
-        <React.StrictMode>
-          <BrowserRouter>
-            <ThemeProvider theme={theme}>
-              <CssBaseline />
-              <Suspense fallback="loading...">
-                <App />
-              </Suspense>
-            </ThemeProvider>
-          </BrowserRouter>
-        </React.StrictMode>
+        {/* <React.StrictMode> */}
```

Comment:
Leave a comment to explain why we commented this code.
Do for all webapp and add reference to the current issue